src/herbivoresAndCarnivores.js

```diff
@@ -1,15 +1,32 @@
 'use strict';
 
 class Animal {
-  // write your code here
+  constructor(name, health = 100) {
+    this.health = health;
+    this.name = name;
+    Animal.alive.push(this);
+  }
 }
 
+Animal.alive = [];
+
 class Herbivore extends Animal {
-  // write your code here
+  constructor(health = 100, name) {
+    super(health, name);
+    this.hidden = false;
+  }
+  hide() {
+    this.hidden = true;
```

Comment:
<img width="730" alt="Screenshot 2022-03-29 at 16 55 53" src="https://user-images.githubusercontent.com/94292654/160627871-75fc2f14-2ac3-46af-aab5-d322e20cb645.png">

when it will be true, it should be changed to false